activerecord/lib/active_record/relation/batches.rb

```diff
@@ -230,9 +230,5 @@ def apply_limits(relation, begin_at, end_at)
       relation = relation.where(table[primary_key].lteq(end_at)) if end_at
       relation
     end
-
-    def batch_order
-      "#{quoted_table_name}.#{quoted_primary_key} ASC"
-    end
```

Comment:
> However, is it really necessary to have the order argument in a separate method?

Someone may have overridden `batch_order` in their model so I'd rather not break their app without a good reason.